src/index.js

```diff
@@ -1,11 +1,40 @@
 import './styles/style.scss';
-import scorelist from './modules/array.js';
+import {
+  url, refresh, form, user, score,
+} from './modules/variables.js';
 
 const userScoreList = document.querySelector('.add-to-board');
 
 userScoreList.innerHTML = '';
-scorelist.forEach((score) => {
-  userScoreList.innerHTML += `
-    <li class="list-item">Name: ${score}</li>
-    `;
+
+const getData = async () => {
+  const response = await fetch(`${url}/games/Zl4d7IVkemOTTVg2fEid/scores/`);
+  const data = await response.json();
+  userScoreList.innerHTML = data.result
+    .map((score) => `<li>${score.user}: ${score.score}</li>`)
+    .join('');
+};
+
+refresh.addEventListener('click', getData);
+
+const setData = async (post) => {
+  await fetch(`${url}/games/Zl4d7IVkemOTTVg2fEid/scores/`, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify(post),
+  });
+};
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const post = {
+    user: `${user.value}`,
+    score: `${score.value}`,
+  };
+  setData(post).then(() => {
+    e.target.reset();
+    getData();
+  });
```

Comment:
- [ ] (Optional) It would be nice to validate the `user` value and the `score` value before submitting data to the API POST endpoint. You can read this [article](https://www.w3schools.com/js/js_validation.asp) to have an idea of how it can be implemented. 👍

Reply:
Hello @Gambit142 Thank you for the review.
I have a question regarding the validation, since I am using the text/number and required attributes in HTML, should I still validate the inputs in JS? or do you mean adding specific type of input using regex?
Thanks again and happy coding 😃

Reply:
Hi @EidHachem, Yes, you can still validate in JS or use regex. It depends on the one you feel more comfortable implementing. 👍